draft-iab-use-it-or-lose-it.md

```diff
@@ -65,15 +65,15 @@ informative:
 
 The ability to change protocols depends on exercising the extension and version
 negotiation mechanisms that support change.  Protocols that don't use these
-mechanisms can find that deploying changes can be difficult and costly.
+mechanisms can find it difficult and costly to deploy changes.
```

Comment:
nit: if we're modifying this perhaps we fix the fact that the protocol isn't the one deploying changes?